libraries/botbuilder-core/botbuilder/core/integration/bot_framework_http_client.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I believe this should just be in `core/`, there isn't a parallel story for this class here in Python compared to C#.